src/components/Dropdown.jsx

```diff
@@ -0,0 +1,29 @@
+import { useState } from 'react';
+import icArrowDown from '../assets/ic_arrow_down.svg';
+import icSort from '../assets/ic_sort.svg';
+import styles from './Dropdown.module.css';
+
+const option = { favorite: "좋아요순", recent: "최신순" }
+
+function Dropdown({ state, setState, mode }) {
+    const [isOpen, setIsOpen] = useState(false);
+
+    return (
+        <button 
+            className={`${styles.dropdown} ${(mode === "Mobile" && styles.small)}`} 
+            onClick={() => setIsOpen((prev) => !prev)} 
+            onBlur={() => setIsOpen(false)}
+        >
+            { mode !== "Mobile" 
+                ? <> <div>{option[state]}</div> <img src={icArrowDown} /> </>
+                : <img src={icSort} /> }
+            { isOpen &&
+                <ul className={styles.dropdownList}>
+                    <li onClick={() => setState("recent")}>{option.recent}</li>
```

Comment:
❗️ 수정요청
이런 클릭 가능한 요소의 경우 버튼을 사용해주세요. 더 적절한 태그가 있는데 이미지 태그에 onClick 을 주게되면 button에게 기대하는 동작을 제대로 수행하지 못합니다!

```suggestion
                    <li>
                      <button type="button" value="recent" onClick={(e) => setState(e.target.value)}>{option.recent}</button>
                    </li>
```

위에처럼 button 태그로 바꾸시고 onClick 시 event 객체에서 button 태그의 value를 받아오는 방식으로 구현하시는 것을 추천드려요! 이렇게 하시면 option 객체를 기준으로 순회하셔서 작성하실수도 있습니다~

Reply:
nested button 문제로 인해 `<input type="button" />`으로 구현하겠습니다!